### DISCUSSION TITLE
Which timezone appwrite function CRON schedule use?

### DISCUSSION THREAD

**Discussion by LDY681** [appwrite/appwrite#5819] [Q&A] [answered]
Hi, I would like to know which timezone does appwrite function use with CRON schedule.  
https://appwrite.io/docs/functions#scheduled-execution
And if there is a way to specify the timezone?
Thanks

**Comment by stnguyen90** [comment, marked as answer]
@LDY681, thanks for raising this! It should be UTC. In general, it's a best practice to use UTC for server-side timezone since you can have multiple servers in multiple locations and it's helpful to standardize them all.
